src/main/java/com/softserveinc/ita/pageobjects/BasePage.java

```diff
@@ -55,10 +55,10 @@ public SearchResultPage performSearch() {
     }
 
     @Step("BasePage: Opened user modal window")
-    public UserModal openUserModalWindow() {
+    public T openUserModalWindow() {
```

Comment:
here I'm not sure, is it used somewhere? if we know that it should open user modal, so probably here should returning page object which is referring to the user

Reply:
removed non-usable method